tests/integration_tests/bank/case.go

```diff
@@ -619,11 +619,10 @@ func getDownStreamSyncedEndTs(ctx context.Context, db *sql.DB, tableName string)
 }
 
 func tryGetEndTs(db *sql.DB, tableName string) (result string, ok bool) {
-	query := fmt.Sprintf("admin show ddl jobs where table_name = '%s'", tableName)
+	query := "admin show ddl jobs where table_name = ?"
 	log.Info("try get end ts", zap.String("query", query))
-
 	var line dataRow
-	row := db.QueryRow(query)
+	row := db.QueryRow(query, tableName)
```

Comment:
What's the difference in this change?

Reply:
the old one may casue sql jnjection see more [here](https://ci.pingcap.net/blue/organizations/jenkins/atom-gosec/detail/atom-gosec/840/tests/)

Reply:
see details https://cwe.mitre.org/data/definitions/89.html